node/tessel-export.js

```diff
@@ -969,11 +971,16 @@ Tessel.SPI = function(params, port) {
 };
 
 Tessel.SPI.prototype.send = function(data, callback) {
-  this._port.cork();
+
   this.chipSelect.low();
-  this._port._tx(data, callback);
-  this.chipSelect.high();
-  this._port.uncork();
+
+  setTimeout(() => {
```

Comment:
Your change isn't quite what I meant. In the case where you don't add a delay, you want to leave the entire transaction in one `cork` / `uncork` pair so it goes out in a single packet, but when you add a delay, you don't need `cork`/`uncork`, because you do want the commands separated.